src/plugins/automod/filters/BadWords.js

```diff
@@ -36,7 +36,7 @@ class BadWordsFilter extends Filter {
         const patternsDesc = describeMatchedPatterns('\nThe following patterns were matched:\n', this.patterns, message.content);
         const postCcnormPatternsDesc = describeMatchedPatterns('\nThe following post-ccnorm patterns were matched:\n', this.postCcnormPatterns, remove(message.content));
 
-        let logMessage = `**Reason**: Bad words matched${patternsDesc}${postCcnormPatternsDesc}\n<@${message.author.id}>`; // TODO: # of offenses
+        let logMessage = `**Reason**: Bad words matched${patternsDesc}${postCcnormPatternsDesc}\n<@${message.author.id}\nThe message:\n${message.content}>`; // TODO: # of offenses
```

Comment:
The message may be over 2000 characters long, can you limit the length so it can be sent?

Reply:
embed descriptions can also go up to 4096 characters. but combined with the static text and interpolations, this may also be exceeded

Reply:
hm maybe "center" it on the part of the message that tripped the filter, that way if the message is cut off it doesn't cut off the part that we care about? Tho how many messages would be over 2000 characters? Plus there's also the log channel so maybe it's not that important that it's "centered"? (Also I'm not sure of a way to limit it nicely, unless you mean like "if it's longer than 2000 characters, then remove everything after it?)

Reply:
Also, Nitro.

Reply:
> hm maybe "center" it on the part of the message that tripped the filter

Sure.

> Tho how many messages would be over 2000 characters?

Does it matter? You don't want your mod log to fail in any case.

> Also I'm not sure of a way to limit it nicely

[`String.prototype.slice`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/slice) on the log message is one way.

Reply:
> Does it matter? You don't want your mod log to fail in any case.

I meant that as in would it matter if the message was trimmed at 2000 characters given that I doubt there's gonna be that many, if any messages that are 2000+ characters.